dev-learning/dev-learning.csproj

```diff
@@ -20,11 +20,11 @@
     <PackageReference Include="Microsoft.VisualStudio.Web.CodeGeneration.Design" Version="3.1.2" />
     <PackageReference Include="MySql.Data" Version="8.0.19" />
     <PackageReference Include="MySql.Data.EntityFrameworkCore" Version="8.0.19" />
+    <PackageReference Include="Pomelo.EntityFrameworkCore.MySql" Version="3.1.1" />
```

Comment:
When I was trying to do this didn't find a lib which helps to connect entity framework to mysql - nice 👍